src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.search {
+  display: block;
+  position: relative;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search__icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
+}
+
+.search__input {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: inherit;
+  font-size: 16px;
+  padding-left: 62px;
+  margin-bottom: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search__input:hover {
+  border: 1px solid #e1e7ed;
```

Comment:
create variables for repeated colors

Reply:
i removed the duplicate lines you mentioned below

this color is no longer repeated in the code